src/AppController.m

```diff
@@ -2496,9 +2496,6 @@ -(void)viewArticlePages:(id)sender inPreferredBrowser:(BOOL)usePreferredBrowser
             }
 		}
 		[self openURLs:urls inPreferredBrowser:usePreferredBrowser];
-		
-		if (!db.readOnly)
-            [articleController markReadByArray:articlesWithLinks readFlag:YES];
 	}
```

Comment:
I didn't intend to include that commit in this pull request. Github seems to just do its own thing.